app/boards/intel_adsp_ace20_lnl.conf

```diff
@@ -70,6 +70,8 @@ CONFIG_WINSTREAM_CONSOLE=n
 
 CONFIG_INTEL_ADSP_IPC=y
 
+CONFIG_PROBE=y
```

Comment:
config PROBE
	bool "Probes enabled"
	default y if CAVS

Should this be CAVS | ACE so that all platform have this by default?

config PROBE_DMA_MAX
	int "Maximum injection probe DMAs attached"
	
The name of the Kconfig is very misleading, consider renaming to make it self-explanatory that this is for INJECTION only.

Reply:
I'm fine with these handled in separate PRs. Especially with vendor defaults, I wonder if it would be better to have some common definition files e.g. for Intel SOF platforms. The PROBE is really a generic feature not tied to hardware in anyway, so having the defaults set based on HW capabilities is a bit misleading. This is really about which feature Intel SOF builds want to have enabled.